doc/sphinx-guides/source/installation/config.rst

```diff
@@ -507,6 +507,10 @@ A Dataverse installation can alternately store files in a Swift or S3-compatible
 
 A Dataverse installation may also be configured to reference some files (e.g. large and/or sensitive data) stored in a web-accessible trusted remote store.
 
+A Dataverse installation can be configured to allow out of band upload by setting the ``dataverse.files.\<id\>.upload-out-of-band`` JVM option to ``true``.
+By default, Dataverse support uploading files via the :ref:`add-file-api`. With S3 stores, a direct upload process can be enabled to allow sending the file directly to the S3 store (without any intermediate copies on the Dataverse server).
+With the upload-out-of-band option enabled, it is also possible for file upload to be managed manually or via third-party tools, with the :ref:`add-file-metadata-api` call used to add metadata and inform Dataverse that a new file has been added to the relevant store.
```

Comment:
The doc build is failing because the add-file-metadata-api ref doesn't exist.

Reply:
I made it such that it is not a 'ref' anymore. I was not sure if I should add some references here. Right now, it just states literally add-file-metadata-api, without it being any reference, etc.